x/ccv/provider/types/msg.go

```diff
@@ -280,13 +280,15 @@ func (msg MsgSetConsumerCommissionRate) ValidateBasic() error {
 // NewMsgCreateConsumer creates a new MsgCreateConsumer instance
 func NewMsgCreateConsumer(submitter, chainId string, metadata ConsumerMetadata,
 	initializationParameters *ConsumerInitializationParameters, powerShapingParameters *PowerShapingParameters,
+	allowlistedRewardDenoms *AllowlistedRewardDenoms,
 ) (*MsgCreateConsumer, error) {
 	return &MsgCreateConsumer{
 		Submitter:                submitter,
 		ChainId:                  chainId,
 		Metadata:                 metadata,
 		InitializationParameters: initializationParameters,
 		PowerShapingParameters:   powerShapingParameters,
+		AllowlistedRewardDenoms:  allowlistedRewardDenoms,
```

Comment:
Shouldn't the `AllowlistedRewardDenoms` be validated?

Reply:
Was partially validated in the handler. Made the validation full.